data/strings/strings.txt

```diff
@@ -10665,7 +10667,7 @@
 
   [editor_report_problem_duplicate_place_title]
     tags = android
-    en = Duplicated place
+    en = Duplicate place
```

Comment:
This is really about a duplicate object in the map database.  Duplicate place sounds like it is about the real world.   This may be hard to change.  The diff above is perfectly ok, so this is really a side comment.

Reply:
I don't think duplicated vs duplicate matters and I don't see an en_US preference for duplicated, or a reason to have en_GB and en_US be different.

What I meant is that the real issue is that there are two map objects representing a single real-world entity.   That's "duplicate map database entry".   There is no duplication *in the real world*, which is what place implies.   I agree that casually many would say that.   I might say "Place appears multiple times on the map" to make that clearer.

Reply:
That is true, but i think this is ok in context. A user wouldn't create a report if there are two identical places on the map that are also there in real life. I reckon it's probably fine :)